build.gradle.kts

```diff
@@ -0,0 +1,35 @@
+import org.hypertrace.gradle.publishing.License.APACHE_2_0
+
+plugins {
+    `java-gradle-plugin`
```

Comment:
Ironically, this file is misformatted 😉  - would it be worth adding kts support to the plugin (or maybe in another PR?)

Reply:
The plugin has `kts` support, but can we bootstrap the plugin's `build.gradle.kts` to format it's own `build.gradle.kts`?

Reply:
so this is always fun + weird with writing plugins. You could publish it locally, use it locally and then remove it. But you'll need to publish it once before it can actually use itself (and then, it'll always be a version behind). At least, I haven't found a way to use your current changes in the PR - maybe some way to do it with buildSrc (where you can define local plugins), but not sure if you can publish from there.